AirCasting/CreateSessionViews/CreateSessionDetailsView.swift

```diff
@@ -145,8 +159,12 @@ private extension CreateSessionDetailsView {
         let fakeLocation = CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
         if isIndoor {
             sessionContext.startingLocation = fakeLocation
+            locationTracker.googleLocation = [PathPoint(location: CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0), measurement: 20.0)]
         } else {
-            sessionContext.obtainCurrentLocation()
+            let lat = (locationTracker.locationManager.location?.coordinate.latitude)!
```

Comment:
are you sure it's safe to force unwrap?

Reply:
I can say that I am never 100% sure, I will handle that